### DISCUSSION TITLE
Creating a website without writing HTML and JavaScript?

### DISCUSSION THREAD

**Discussion by ChronoDK** [fastapi/fastapi#2876] [Questions]
I have created a backend with FastAPI, and now I need a simple frontend to interact with the backend. Simple stuff like listing entries in the database, and adding entries. I don't care much about how the site looks. Is there a simple library that allows me to create the website, without writing any HTML/CSS/JavaScript and then serving templates. Just pure Python. Is there a tool like that out there somewhere?

**Comment by ChronoDK** [comment]
I found something called JustPy. It looks pretty neat, has anyone tried it with FastAPI?
https://github.com/elimintz/justpy

Also, Streamlit.io does some similar things, but seems focused on data visualization and looks complicated to use with FastAPI?

**Comment by Ro0tk1t** [comment]
looks good
